modules/system-info/reporters/user.php

```diff
@@ -15,6 +15,10 @@
  */
 class User extends Base {
 
+	public function is_enabled() {
+		return (bool) count( (array) wp_get_current_user()->data );
```

Comment:
`wp_get_current_user` returns `0`  if the user isn't logged-in.
```suggestion
		return wp_get_current_user();
```

Reply:
Handled.